WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift

```diff
@@ -0,0 +1,250 @@
+import UIKit
+import WordPressShared
+
+
+// MARK: - LoginEpilogueTableViewController
+//
+class LoginEpilogueTableViewController: UITableViewController {
+
+    /// TableView's Datasource
+    ///
+    private let blogDataSource = BlogListDataSource()
+
+    /// Epilogue Metadata
+    ///
+    private var epilogueUserInfo: LoginEpilogueUserInfo? {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    /// Site that was just connected to our awesome app.
+    ///
+    private var credentials: WordPressCredentials?
+
+
+    // MARK: - Initializers
+
+    required init?(coder aDecoder: NSCoder) {
```

Comment:
Arg. I forgot to mark this in my review.  We could probably nuke this since all it does is call super.